### PR TITLE
Remove 'support.google.com' references from documentation.

### PR DESCRIPTION
**Pull request by tst-eclamar**
Only half of the pages with references to 'support.google.com' are updated in this push.  The rest will be written up in a new bug.

### CODE REVIEW

**Comment by tst-lsavoie** [comment]
Fixes #148.

File: earth_enterprise/docs/landing_page/manual/answer/6004748.html [hunk 1]
```diff
@@ -286,7 +286,7 @@
 </td>
 <td>Localize international URL for support answers
 </td>
-<td><code>https://support.google.com/earth/#topic=4363013</code>
```
Comment (tst-lsavoie) [inline earth_enterprise/docs/landing_page/manual/answer/6004748.html]: There are other configuration values on this page that point to google.com domains (like www.google.com and earth.google.com).  Can we change those to reference opengee.org or something on the github wiki?

**Comment by tst-eclamar** [comment]
Sure, but I don't know the opengee.org or github wiki well enough yet to
know what these values should be.

There are 5 references to google on this webpage(6004748.html), three of
which are the same.

Any recommendations will be appreciated.

Thanks,
Eric.

**Comment by tst-lsavoie** [comment]
Since there's not a good way to fix 6004748.html and it's a fairly minor part of the documentation, we decided to leave it as is and fix it on a future issue.

**Pull request merged**